index.js

```diff
@@ -4,10 +4,24 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "red"
 
 /***** Deliverable 3 *****/
-// console.log("PLAYERS array looks like this:", PLAYERS)
+console.log("PLAYERS array looks like this:", PLAYERS)
 
+PLAYERS.forEach(function(player) {
+    let div = document.createElement("div")
+    div.className = "player"
+    div.dataset.number = `${player.number}`
```

Comment:
Looks good, small note - string interpolation isn't necessary since `player.number` is already a string!

```js
div.dataset.number = player.number}
``